src/main/java/io/stargate/sgv2/jsonapi/service/resolver/UpdateOneCommandResolver.java

```diff
@@ -80,11 +80,21 @@ public Class<UpdateOneCommand> getCommandClass() {
   public Operation resolveTableCommand(
       CommandContext<TableSchemaObject> ctx, UpdateOneCommand command) {
 
-    return new UpdateTableOperation(
-        ctx,
-        tableUpdateResolver.resolve(ctx, command),
+    var builder =
+        new UpdateAttemptBuilder<>(ctx.schemaObject(), tableUpdateResolver.resolve(ctx, command));
```

Comment:
the output of the udpateResolver is the update clause, lets move so we pass the update clause to  build() function for the operationattempt builder

Reply:
fixed